scripts/cleanFolders.js

```diff
@@ -12,7 +12,7 @@ function copyImage(folderImage, imagePath, outputSource) {
  * @param {object} data
  */
 
-module.exports = (inputSource, outputSource, data) => {
+module.exports = (inputSource, outputSource, parseMode, data) => {
```

Comment:
Why can't you detect locale folder without having to pass this extra parameter?

Reply:
It can be detected, but this parameter only specifies if the search should be done only in the "default" folder (as it's currently happening) or considering all folders in the input path.